pkg/codegen/internal/test/testdata/other-owned/schema.json

```diff
@@ -0,0 +1,275 @@
+    },
+    "bar::BarResource": {
+      "isComponent": true,
+      "properties": {
+        "foo": {
+          "$ref": "#/resources/example::Resource"
+        }
+      },
+      "inputProperties": {
+        "foo": {
+          "$ref": "#/resources/example::Resource"
+        }
+      },
+      "type": "object"
+    },
+    "foo::FooResource": {
+      "isComponent": true,
+      "properties": {
+        "foo": {
+          "$ref": "#/resources/example::Resource"
+        }
+      },
+      "inputProperties": {
+        "foo": {
+          "$ref": "#/resources/example::Resource"
+        }
+      },
+      "type": "object"
+    },
+
+    "example::OverlayResource": {
+      "isOverlay": true,
+      "properties": {
+        "foo": {
+          "$ref": "#/types/example::ConfigMapOverlay"
+        },
+        "bar": {
+          "$ref": "#/types/example::EnumOverlay"
+        }
+      },
+      "inputProperties": {
+        "foo": {
+          "$ref": "#/types/example::ConfigMapOverlay"
+        },
+        "bar": {
+          "$ref": "#/types/example::EnumOverlay"
+        }
+      },
+      "type": "object"
+    },
+    "example::TypeUses": {
+      "properties": {
+        "foo": {
+          "$ref": "#/types/example::Object"
+        },
+        "bar": {
+          "$ref": "#/types/example::SomeOtherObject"
+        },
+        "baz": {
+          "$ref": "#/types/example::ObjectWithNodeOptionalInputs"
+        }
+      },
+      "inputProperties": {
+        "foo": {
+          "$ref": "#/types/example::Object"
+        },
+        "bar": {
+          "$ref": "#/types/example::SomeOtherObject"
+        },
+        "baz": {
+          "$ref": "#/types/example::ObjectWithNodeOptionalInputs"
+        }
+      },
+      "type": "object"
+    }
+  },
+  "functions": {
+    "example::argFunction": {
+      "inputs": {
+        "properties": {
+          "arg1": {
+            "$ref": "#/resources/example::Resource"
+          }
+        }
+      },
+      "outputs": {
+        "properties": {
+          "result": {
+            "$ref": "#/resources/example::Resource"
+          }
+        }
+      }
+    },
+    "example::overlayFunction": {
+      "isOverlay": true,
+      "inputs": {
+        "properties": {
+          "arg1": {
+            "$ref": "#/resources/example::Resource"
+          }
+        }
+      },
+      "outputs": {
+        "properties": {
+          "result": {
+            "$ref": "#/resources/example::Resource"
+          }
+        }
+      }
+    }
+  },
+  "language": {
+    "csharp": {
+      "projectReferences": [
+        "..\\..\\..\\..\\..\\..\\..\\sdk\\dotnet\\Pulumi\\Pulumi.csproj"
+      ],
+      "packageReferences": {
+        "Pulumi": "3.13"
+      },
+      "namespaces": {
+        "Pulumi": "Other"
+      }
+    },
+    "go": {
+      "importBasePath": "github.com/pulumi/pulumi/pkg/v3/codegen/internal/test/testdata/other-owned/go/example",
+      "generateExtraInputTypes": true
+    },
+    "nodejs": {
```

Comment:
Would it be hard to skip irrelevant languages for test cases that add no value for a given test case (all languages except C# in this case)?

Reply:
Not really. I will add a skipping feature.